### ISSUE TITLE
Paper pdf not rendering and bibliography empty

### ISSUE THREAD

**Issue by AnjaConev** [suchanek/proteusPy#9]
Hi, great work and thank you for addressing all my comments so far!

Here is my final issue for this review -  two things:

1. I am not able to open the paper pdf in `proteusPy/paper
/paper.pdf` - I get an error that the pdf is invalid (both in Firefox, Chrome, on GitHub).

2. When I checked out `proteusPy/paper/paper.md` I noticed that the bibliography section is empty - it might be just a randering issue though.

**Comment by suchanek** [comment]
Ugg that’s weird. Thanks for catching. I’m abroad and don’t have my laptop but I’ll see if I can figure it out. Best,Eric G. Suchanek, PhD Astronomer at largeMonterey Institute for Research in Astronomy200 Eighth Street, Marina CA 93933(831-883-1000) Fax: 883-1031Dedicated to Research and Education in Astronomy for 50 Years.On Apr 2, 2024, at 12:23 PM, Anja Conev ***@***.***> wrote:﻿
Hi, great work and thank you for addressing all my comments so far!
Here is my final issue for this review -  two things:


I am not able to open the paper pdf in proteusPy/paper /paper.pdf - I get an error that the pdf is invalid (both in Firefox, Chrome, on GitHub).


When I checked out proteusPy/paper/paper.md I noticed that the bibliography section is empty - it might be just a randering issue though.
